src/utils/fetch.js

```diff
@@ -31,10 +31,16 @@
 /*
  * Creates URL request path
  */
-const encodeQueryData = data => Object.keys(data)
-  .filter(k => typeof data[k] !== 'object')
-  .map(k => encodeURIComponent(k) + '=' + encodeURIComponent(data[k]))
-  .join('&');
+const encodeQueryData = (data, nesting = '') => {
+  const pairs = Object.entries(data).map(([key, val]) => {
+    if (typeof val === 'object') {
```

Comment:
FYI, in JavaScript `null` is actually an object. So this will crash in those instances :)

Reply:
I think what one could do here is to always add a `.<type>` to the keys :thinking: 

Example:

```
{
  date: 12345678,
  name: 'anders',
  active: true,
  metadata: {
    something: null
  }
}
```

```
date.i=12345678&name.s=anders&active.b=true&metadata.something.n=null
```

This way it's pretty easy to get the correct types:

```js
const typeMap = {
  i => str => parseInt(i, 10),
  s => str => str,
  b => str => str === 'true',
  n = > str => null,
}
```